lib/admin/actions/admin.js

```diff
@@ -87,44 +96,14 @@ export function createUser (credentials: any) {
   }
 }
 
-export function deletingUser (user: UserProfile) {
-  return {
-    type: 'DELETING_USER',
-    user
-  }
-}
-
-export function deletedUser (user: UserProfile) {
-  return {
-    type: 'DELETED_USER',
-    user
-  }
-}
-
 // server call
 export function deleteUser (profile: UserProfile) {
   return function (dispatch: dispatchFn, getState: getStateFn) {
-    dispatch(deletingUser(profile))
     const url = `/api/manager/secure/user/${profile.user_id}`
     return dispatch(secureFetch(url, 'delete'))
       .then(response => response.json())
       .then(result => {
-        dispatch(deletedUser(profile))
```

Comment:
If these actions are only for debugging purposes, there should at least be a way to disable them in production.  Furthermore, if they are for debugging, we should be using a logging tool, not dispatching actions because they happen to cause a logging statement to be printed.